charts/gateway-helm/templates/envoy-gateway-service.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
   labels:
     control-plane: envoy-gateway
   {{- include "eg.labels" . | nindent 4 }}
+  annotations:
```

Comment:
Sorry, I may not clear, this should be on deployment/pod instead of service?

Reply:
I set it wrong, it should be on the pod. forgive me

Reply:
what I learn from other projects is on Deployment/Pod, what the offical document said?

Reply:
There is no official recommendation on which one to use. However, when only capturing a small number of instances such as eg-controller, it may be better to set the annotation to Deployment/Pod.
In addition: when crawling a group of stateless applications with a large base, setting annotations to Service is indeed a common behavior :)

Reply:
I will change